ansible/roles/debops.docker_server/defaults/main.yml

```diff
@@ -13,119 +13,119 @@
 # Docker packages and installation [[[
 # ------------------------------------
 
-# .. envvar:: docker__distribution [[[
+# .. envvar:: docker_server__distribution [[[
 #
 # The OS distribution which is used to select upstream APT repository.
-docker__distribution: '{{ ansible_local.core.distribution
+docker_server__distribution: '{{ ansible_local.core.distribution
                           if (ansible_local|d() and ansible_local.core|d() and
                               ansible_local.core.distribution|d())
                           else ansible_distribution }}'
```

Comment:
It would look nicer if the values were indented to the new variable name lengths, here and in other such cases.